modules/Messenger/messenger_postProcess.php

```diff
@@ -51,47 +51,22 @@
         $_POST = $validator->sanitize($_POST, ['body' => 'HTML']);
 
 		//Validate Inputs
-		$email=$_POST["email"] ;
-		if ($email!="Y") {
-			$email="N" ;
-		}
+		$email=$_POST["email"] ?? 'N';
```

Comment:
The original code would set `$email` to `'N'` if it is anything other than `'Y'`. The new code would only set `$email` to `'N'` if it is not set. This could introduce new logical errors.